src/utilities/oas-operation/oas-operation.factory.ts

```diff
@@ -15,16 +16,23 @@ const OPERATION_KEYS = [
 class OASOperationFactory {
   public static buildFromPaths(paths: PathsObject): OASOperation[] {
     const pathObjects = Object.values(paths);
-
+    let params: ParameterObject[] = [];
     const operationObjects = pathObjects.flatMap((path) => {
       return Object.entries(path).reduce<OperationObject[]>(
-        (operations, [key, object]) =>
-          OPERATION_KEYS.includes(key) ? [...operations, object] : operations,
+        (operations, [key, object]) => {
+          if (key === 'parameters') params = [...object];
```

Comment:
Not sure if there's a standard with Javascript, but are if-statement blocks usually wrapped in curly braces or are one-liners like this ok?

Reply:
yup, one-liners like this are ok